pkg/util/initsystem/initsystem.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 type InitSystem interface {
+	// return a string describing how to enable a service
+	EnableCommand(service string) string
```

Comment:
Why are we adding this one? Is it going to be used somewhere?

Reply:
It's not. These methods are added by this PR. This is the content of the latest master:
https://github.com/kubernetes/kubernetes/blob/master/pkg/util/initsystem/initsystem.go